docs/notebooks/index.rst

```diff
@@ -1,12 +1,46 @@
-========
-Examples
-========
+=========
+Notebooks
+=========
+
+These notebooks demonstrate a few features of the Ravenpy package when integrated into the PAVICS-Hydro service or run locally.
+
+If you're unfamiliar with notebooks, note that typing `TAB` after an object will display a drop-down menu of the object's attributes and methods, and that you need to either press the "run" button in the top of the JupyterLab window on the PAVICS server, or "hit `CTRL-Enter` to run a *cell*. You can also type `?` after a function or method to display the corresponding help message. See more details in the first Notebook below (00 - Introduction to JupyterLab).
+
+Getting started - Tutorial
+==========================
+
+.. toctree::
+   :maxdepth: 1
+
+   00_Introduction_to_JupyterLab
+   01_Getting_watershed_boundaries
+   02_Extract_geographical_watershed_properties
+   03_Extracting_forcing_data
+   04_Emulating_hydrological_models
+   05_Advanced_RavenPy_configuration
+   06_Raven_calibration
+   07_Making_and_using_hotstart_files
+   08_Getting_and_bias_correcting_CMIP6_data
+   09_Hydrological_impacts_of_climate_change
+   10_Data_assimilation
+   11_Climatological_ESP_forecasting
+   12_Performing_hindcasting_experiments
```

Comment:
Maybe just something of interest, but you can provide different titles for these using:
```
My Custom Title <00_Introduction_to_JupyterLab>
```